cmd/psqldef/tests.yml

```diff
@@ -984,6 +984,17 @@ RemoveDefaultExpression:
     );
   output: |
     ALTER TABLE "public"."users" ALTER COLUMN "key" DROP DEFAULT;
+ChangeDefaultExpressionWithAddition:
+  current: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '1 day'::interval)
+    )
+  desired: |
+    CREATE TABLE public.foo (
+      expires_at timestamp with time zone NOT NULL DEFAULT (CURRENT_TIMESTAMP + '3 day'::interval)
+    )
+  output: |
+    ALTER TABLE "public"."foo" ALTER COLUMN "expires_at" SET DEFAULT current_timestamp + '3 day';
```

Comment:
I don't know why `::interval` is dropped. The ALTER TABLE command seems to work anyway. 🤷‍♀️

Reply:
I figured out why. Will upload a fixed version soon.